### ISSUE TITLE
Can provide a download address for pre training weights

### ISSUE THREAD

**Issue by WesternTrail** [wgcban/Metric-CD#2]
Thank you for your work,！and may I ask if the following uses are included in your code
`         self.blocks.load_state_dict(torch.load("/media/lidan/ssd/Chaminda/change_detection/VGG_finetune_AID/saved_model/vgg16_AID_049fc.pth"), strict=False)
`
Can you provide a download address for reading the pre training weight code.

**Comment by abbanbhan** [comment]
Is there any update on this?

**Comment by wgcban** [comment]
model_urls = {
    'vgg11': 'https://download.pytorch.org/models/vgg11-bbd30ac9.pth',
    'vgg13': 'https://download.pytorch.org/models/vgg13-c768596a.pth',
    'vgg16': 'https://download.pytorch.org/models/vgg16-397923af.pth',
    'vgg19': 'https://download.pytorch.org/models/vgg19-dcbb9e9d.pth',
    'vgg11_bn': 'https://download.pytorch.org/models/vgg11_bn-6002323d.pth',
    'vgg13_bn': 'https://download.pytorch.org/models/vgg13_bn-abd245e5.pth',
    'vgg16_bn': 'https://download.pytorch.org/models/vgg16_bn-6c64b313.pth',
    'vgg19_bn': 'https://download.pytorch.org/models/vgg19_bn-c79401a0.pth',
}

**Comment by wgcban** [comment]
@abdulbb @WesternTrail

**Comment by wgcban** [comment]
https://github.com/minar09/VGG16-PyTorch/blob/master/vgg.py
